src/BuzzHttpAdapter.php

```diff
@@ -47,6 +47,8 @@ public function __construct(Browser $browser = null, ConfigurationInterface $con
             );
         }
 
+        $browser->getClient()->setVerifyPeer($configuration->getSSLVerifyPeer());
```

Comment:
Yes it is a client setting and should only be called once in theory but I can't predict what end users will do with for example here, the internal client and since its a reference, then, people can update its state between initialization and execution. This is why I prefer set it at runtime instead of construction time. I have applied the same logic to the other configuration options.